server/src/weaverbird/backends/mongo_translator/steps/todate.py

```diff
@@ -0,0 +1,306 @@
+from typing import List
+
+from weaverbird.backends.mongo_translator.steps.types import MongoStep
+from weaverbird.pipeline.steps import ToDateStep
+
+
+def translate_todate(step: ToDateStep) -> List[MongoStep]:
```

Comment:
Perhaps we could split this function in smaller ones ?

Reply:
:+1:

Reply:
[33df3e3](https://github.com/ToucanToco/weaverbird/pull/1285/commits/33df3e3bfb13cb40a39956d5038418133eb9267e)